examples/canjs/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+!node_modules/canjs-localstorage/can.localstorage.js
+!node_modules/canjs/can.jquery.js
+!node_modules/jquery/dist/jquery.js
+!node_modules/todomvc-common/base.js
+!node_modules/todomvc-app-css/index.css
+!node_modules/todomvc-common/base.css
```

Comment:
For instance

https://github.com/tastejs/todomvc/pull/1158/files#diff-a7e35d76a245875ebca50c7572724d29R1
https://github.com/tastejs/todomvc/pull/1154/files#diff-56a795cbc928ccf2c01f4612d6983392R1
https://github.com/tastejs/todomvc/pull/1157/files#diff-cc579ff3e283db4e7f4311da9440fc8eR1

are all is incorrect

https://github.com/tastejs/todomvc/pull/1149/files#diff-2b72fb0b4579f216ecf9635cb679d794R7
https://github.com/tastejs/todomvc/pull/1134/files#diff-252f122c2b5be2db6901ad001e01392dR1

are correct (however can be instantly invalidated if npm decided to add any new "private" folders such as .bin)

meh just some thoughts